test/emulator_backend.go

```diff
@@ -107,14 +107,17 @@ type keyInfo struct {
 	signer     crypto.Signer
 }
 
+var Chain = flow.MonotonicEmulator.Chain()
+
+var CommonContracts = emulator.NewCommonContracts(Chain)
```

Comment:
I guess these two variables `Chain` and 1CommonContracts` don't need to be exported (can be lower-case)

Reply:
Oh, you are right. The EmulatorBackend is in the same package, so not export is needed indeed. Will update

Reply:
Fixed in https://github.com/onflow/cadence-tools/pull/197/commits/b6814fa5cdf297ee802657a102a1c4a53aa959c7.
Along with other types/functions that were exported without reason.